numpy/core/tests/test_regression.py

```diff
@@ -2080,6 +2080,22 @@ def test_deepcopy_on_0d_array(self):
         self.assertTrue(arr is not arr_cp)
         self.assertTrue(isinstance(arr_cp, type(arr)))
 
+    def test_deepcopy_F_order_object_array(self):
+        # Ticket #6456.
+        a = {'a': 1}
+        b = {'b': 2}
+        arr = np.array([[a, b], [a, b]], order='F')
+        arr_cp = copy.deepcopy(arr)
+
+        assert_equal(arr, arr_cp)
+        self.assertTrue(arr is not arr_cp)
+        # Ensure that we have actually copied the item.
+        self.assertIsNot(arr[0, 1], arr_cp[1, 1])
```

Comment:
`assertIs` and `assertIsNot` are not available in Python 2.6, use `assertTrue` and `assertFalse` with proper arguments.